core/src/main/java/org/apache/accumulo/core/client/impl/Namespace.java

```diff
@@ -0,0 +1,42 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.apache.accumulo.core.client.impl;
+
+import org.apache.accumulo.core.client.Instance;
+
+public class Namespace {
```

Comment:
> I'm reticent to include structure on the promise of "code getting finished"

With Accumulo being such a complex and mature project, I think incremental code changes should be encouraged.  It is very difficult to make significant code changes all at once.  

> this just seems like a layer of indirection that causes more typing

If useful code can be commited, while at the same time creating opportunity to assist in future significant development, I think it is definitely worth typing the extra "."

Should I add comments to the javadoc explaining how the class is also a placeholder for future work?